client/src/pages/data/scenarios/[scenarioId]/interventions/new.tsx

```diff
@@ -54,7 +54,11 @@ const CreateInterventionPage: React.FC = () => {
       <div className="grid grid-cols-12 gap-6">
         <div className="col-span-10 col-start-2 xl:col-start-3 xl:col-span-8">
           <h1 data-testid="page-title">New intervention</h1>
-          <InterventionForm onSubmit={handleSubmit} isSubmitting={createIntervention.isLoading} />
+          <InterventionForm
+            onSubmit={handleSubmit}
+            isCreation
```

Comment:
It works, and It's ok. But did you consider instead of having `isCreation` something more explicit?? I'd like something like `selectCheckedStrategy="CHILD"`.

Reply:
I'd love to have that as a prop (something like `selectProps` or similar as you said). However, `checkedStrategy="CHILD"` only applies to the business unit selector, so a single prop like that wouldn't suffice.

If the select strategy has to be applied to ALL the selects, then that's another thing and I would gladly change it.